starter-core/src/main/java/io/micronaut/starter/feature/micrometer/MicrometerFeature.java

```diff
@@ -52,4 +57,9 @@ public boolean supports(ApplicationType applicationType) {
     public String getCategory() {
         return Category.MANAGEMENT;
     }
+
+    @Override
+    public String getTitle() {
+        return Arrays.stream(StringUtils.tokenizeToStringArray(getName(), "-")).map(NameUtils::capitalize).collect(Collectors.joining(" "));
```

Comment:
We already have `NameUtils.getNaturalName`. Does that not fit this use case?

Reply:
I can't find the method `getNaturalName` neither on `NameUtils` nor on `StringUtils`